src/app/app-extras.module.ts

```diff
@@ -3,17 +3,33 @@ import {
 } from '@angular/core';
 
 import {
-  SkySampleModule
+  SkyIconModule
+} from '@skyux/indicators';
+
+import {
+  SkyTilesModule
+} from '@skyux/tiles';
+
+import {
+  SkyInlineFormAdapterService,
+  SkyInlineFormModule
 } from './public';
 
+import {
+  SkyTileDemoTileComponent
+} from './visual/inline-form/inline-form-demo-tile.component';
+
 @NgModule({
-  imports: [
-    SkySampleModule
-  ],
   exports: [
-    SkySampleModule
+    SkyIconModule,
+    SkyInlineFormModule,
+    SkyTilesModule
+  ],
+  providers: [
+    SkyInlineFormAdapterService
```

Comment:
Why is this being provided in the `app-extras.module.ts`? I believe it is already provided by the `SkyInlineFormModule`?

Reply:
Correct. It's been removed.